pinot-spi/src/main/java/org/apache/pinot/spi/filesystem/LocalPinotFS.java

```diff
@@ -85,7 +85,7 @@ public boolean doMove(URI srcUri, URI dstUri)
   @Override
   public boolean copy(URI srcUri, URI dstUri)
       throws IOException {
-    copy(toFile(srcUri), toFile(dstUri));
+    copy(toFile(srcUri), toFile(dstUri), false);
```

Comment:
the behavior of this API is not the same in LocalPinotFs vs. S3PinotFs. 

should we also expose a `copy(URI srcUri, URI dstUri, boolean isRecursive)`  ?

Reply:
Why?

Wen there is a need, we can add it. In general, discourage recursive copy, since we know not what is being copied.

Reply:
@mcvsubbu any additional comments? I made a PR based on my idea above #8200?